src/Symfony/Component/PropertyInfo/Extractor/ReflectionExtractor.php

```diff
@@ -348,25 +350,27 @@ public function getWriteInfo(string $class, string $property, array $context = [
         [$accessible, $methodAccessibleErrors] = $this->isMethodAccessible($reflClass, '__set', 2);
         if ($accessible) {
             return new PropertyWriteInfo(PropertyWriteInfo::TYPE_PROPERTY, $property, PropertyWriteInfo::VISIBILITY_PUBLIC, false);
-        } else {
-            $errors = array_merge($errors, $methodAccessibleErrors);
         }
 
-        [$accessible, $methodAccessibleErrors] = $this->isMethodAccessible($reflClass, '__call', 2);
```

Comment:
No need to compute method is accessible nor adding errors if the magic `__call` isn't allowed, right?